packages/fhir-converter/src/templates/cda/Resources/AllergyIntolerance.hbs

```diff
@@ -48,25 +48,32 @@
             {
                 "substance":{{>DataType/CodeableConcept.hbs code=allergyEntry.participant.participantRole.playingEntity.code}},
 
-                {{#each (toArray allergyEntry.entryRelationship)}}
-                    
-                    {{#if (contains (toJsonString this.observation.templateId) '"2.16.840.1.113883.10.20.22.4.9"')}}
-                        "manifestation":
-                        [
-                            {{>DataType/CodeableConcept.hbs code=this.observation.value}},
-                        ],
-                        "onset": "{{formatAsDateTime this.observation.effectiveTime.low.value}}", 
-                    {{/if}}
-                    
-                    {{#if (contains (toJsonString this.observation.templateId) '"2.16.840.1.113883.10.20.22.4.8"')}}
-                        {{#if this.observation.value.translation}}
-                            {{>ValueSet/AllergySeverity.hbs sev=this.observation.value.translation.displayName}}
-                        {{else}}
-                            {{>ValueSet/AllergySeverity.hbs sev=this.observation.value.displayName}}
-                        {{/if}}
-                    {{/if}}
-                    
-                {{/each}}
+                {{#if allergyEntry.entryRelationship }}
+                  {{#each (toArray allergyEntry.entryRelationship)}}
+
+                      {{#if (contains (toJsonString this.observation.templateId) '"2.16.840.1.113883.10.20.22.4.9"')}}
+                          "manifestation":
+                          [
+                              {{>DataType/CodeableConcept.hbs code=this.observation.code}},
+                          ],
+                          "onset": "{{formatAsDateTime this.observation.effectiveTime.low.value}}",
+                      {{/if}}
+
+                      {{#if (contains (toJsonString this.observation.templateId) '"2.16.840.1.113883.10.20.22.4.8"')}}
+                          {{#if this.observation.value.translation}}
+                              {{>ValueSet/AllergySeverity.hbs sev=this.observation.value.translation.displayName}}
+                          {{else}}
+                              {{>ValueSet/AllergySeverity.hbs sev=this.observation.value.displayName}}
+                          {{/if}}
+                      {{/if}}
+
+                  {{/each}}
+                {{else}}
+                  "manifestation":
+                  [
+                      {{>DataType/CodeableConcept.hbs text='noKnownManifestation'}},
+                  ],
+                {{/if}}
```

Comment:
general feedback - let's make "unknown" things to spec, and standardized with our mapping so that our customers can have a consistent way of determining something is known

suggestion is to use `UNK` in https://terminology.hl7.org/1.0.0/ValueSet-v3-Unknown.html for all CodeableConcepts / Codings that we don't have enough info on to fill out